www/front_src/src/Resources/index.tsx

```diff
@@ -97,7 +97,8 @@ const Resources = (): JSX.Element => {
         setListing(retrievedListing);
       })
       .catch((error) => {
-        showError(error.message);
+        setListing(undefined);
+        showError(error?.response?.data?.message || error.message);
```

Comment:
We need to have a consistent way of displaying error messages because right now the behavior is different everywhere.

Reply:
as we discussed, the error format sent by backend will be probably like that : 
`{"code":500,"message":"Error while searching for resources"}`